src/views/meeting.js

```diff
@@ -31,24 +34,61 @@ export class Meeting extends React.Component {
         this._onMeetingLeave = this._onMeetingLeave.bind(this);
     }
 
+    /**
+     * Redirects back the home view if no valid meeting url has been detected.
+     *
+     * @inheritdoc
+     */
+    componentDidMount() {
+        if (!this._getMeetingUrl()) {
+            this._onMeetingLeave();
+        }
+    }
+
     /**
      * Implements React's {@link Component#render()}.
      *
      * @inheritdoc
      */
     render() {
-        // TODO: handle the error case where no meeting name is present
+        const meetingUrl = this._getMeetingUrl();
+
+        if (!meetingUrl) {
+            return null;
+        }
 
         return (
             <View name = 'meeting'>
                 <MeetingFrame
                     displayName = { this.props.displayName }
-                    meetingName = { this.props.match.params.name }
+                    meetingUrl = { meetingUrl }
                     onMeetingLeave = { this._onMeetingLeave } />
             </View>
         );
     }
 
+    /**
+     * Parses the current window location for a valid meeting url. The value
+     * {@code null} will be returned if no valid meeting url is found.
+     *
+     * @private
+     * @returns {string|null}
+     */
+    _getMeetingUrl() {
+        const queryParams = new URLSearchParams(this.props.location.search);
+        const meetingUrl = queryParams.get('location');
+
+        if (!meetingUrl) {
+            return null;
+        }
+
+        if (isValidMeetingUrl(meetingUrl, VALID_MEETING_HOSTS)) {
```

Comment:
should `isValidMeetingUrl` just handle null as well?

Reply:
It returns false for null, if that answers your question. The explicit return of null above is to handle an error case where somehow the user landed on the /meeting route without a meeting defined (a case that so far can only be reproduced by manually going to the /meeting url).